types/stripe/index.d.ts

```diff
@@ -4979,6 +5021,59 @@ declare namespace Stripe {
         }
     }
 
+    namespace reviews {
+        interface IReview extends IResourceObject {
+            /**
+             * Unique identifier for the object.
+             */
+            "id": string;
```

Comment:
This is strictly not necessary as it is already present from the IResourceObject interface.